Lesson1/MyPlayground.playground/Contents.swift

```diff
@@ -1,2 +1,73 @@
-import UIKit
+var inputData: [Int] = [1, 2, 4, 1]
 
+/// The mean for an array of integers
+/// - Parameter inputData:The array of integers with an amount greater than 2
+/// - Returns: The mean; otherwise, nil.
```

Comment:
For now this is not a showstopper, but what does it mean - "otherwise"? You should give a clear understanding, when _exactly_ the method returns `nil`.

Reply:
Here and below.

Reply:
fixed